roles/openshift_certificate_expiry/test/test_fakeopensslclasses.py

```diff
@@ -1,82 +1,79 @@
-#!/usr/bin/env python
 '''
  Unit tests for the FakeOpenSSL classes
 '''
 
 import os
+import subprocess
 import sys
-import unittest
+
 import pytest
 
-# Disable import-error b/c our libraries aren't loaded in jenkins
-# pylint: disable=import-error,wrong-import-position
-# place class in our python path
-module_path = os.path.join('/'.join(os.path.realpath(__file__).split(os.path.sep)[:-1]), 'library')
+module_path = os.path.join('/'.join(os.path.realpath(__file__).split('/')[:-2]), 'library')
```

Comment:
I doesn't make sense to use `os.path.join` and `'/'.join` together ;-)

Suggestion:

```python
sys.path.insert(1, os.path.realpath(os.path.join(__file__, os.pardir, os.pardir, 'library')))
```